packages/web-components/src/components/va-radio-option/va-radio-option.scss

```diff
@@ -44,6 +45,8 @@
   line-height: 1.4rem;
   margin-right: 1rem;
   width: 1.4rem;
+  min-width: 1.4rem;
+  max-width: 1.4rem;
```

Comment:
Forcing the width was required because without it, the circle became squished

![2 radio options with the second containing 3 lines of wrapped text, but the radio symbol is horizontally compressed making it an oval shape](https://user-images.githubusercontent.com/136959/221910122-ca70a926-f25b-4177-b1d2-7818b517e2ef.png)